vignettes/DataSource.Rmd

```diff
@@ -146,10 +146,19 @@ For both `pressure` and `poci`: <br />
 `psi2pa`: converts psi in Pascal <br />
 `None`: no conversion <br />
 
-### Test data set
+### Test data sets
 
-A `test_dataset` is provided with the `StormR` package. This test data set
+Two `test_dataset` are provided with the `StormR` package (a NetCDF and a CSV file). These test data sets
 comprises the track data of nine storms that occurred near Vanuatu and New
 Caledonia between 2015-2016 and 2020-2021, respectively.
 
+### CSV Formatting
+
+In order to load and read a CSV file in `defStormsDataset()`, it must stick to the following rules:<br />
+* As for the NetCDF file, the CSV file must contains at least a name, season, iso times, longitude, latitude,and a maximum wind speed column.
+* Every observation (row) in the CSV must be sorted in increasing order according to the ISO times AND joined by storm. For example, if several storms happened during the same iso Times, the observations for the storm that first appeared should be stacked before the observations of the second one and so on.
+
+In addition to these mandatory rules above, the default separator is set to a single comma ','. Nevertheless th user might decide to change it using the `sep` parameter in `defStormsDataset()`.
+
```

Comment:
Should include an example on how to access the sample dataset, and also a header of the file so that the user can see the link between the header of the file and the `fields` definition

Reply:
Done